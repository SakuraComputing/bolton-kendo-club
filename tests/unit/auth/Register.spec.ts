import { shallowMount } from '@vue/test-utils'
import Register from '@/views/auth/Register.vue'

describe('Register.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Register);
    expect(wrapper).toMatchSnapshot()
  })
})
