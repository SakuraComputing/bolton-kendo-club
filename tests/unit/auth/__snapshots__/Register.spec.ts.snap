// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register.vue renders correctly 1`] = `
<div>
  <h4>Register</h4>
  <form><label for="name">Name</label>
    <div><input id="name" type="text" required="required" autofocus="autofocus"></div> <label for="email">E-Mail Address</label>
    <div><input id="email" type="email" required="required"></div> <label for="password">Password</label>
    <div><input id="password" type="password" required="required"></div> <label for="password-confirm">Confirm Password</label>
    <div><input id="password-confirm" type="password" required="required"></div>
    <div><button type="submit">Register</button></div>
  </form>
</div>
`;
